examples/java/src/main/java/glide/examples/StandaloneExample.java

```diff
@@ -0,0 +1,125 @@
+/** Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.examples;
+
+import glide.api.GlideClient;
+import glide.api.logging.Logger;
+import glide.api.models.configuration.GlideClientConfiguration;
+import glide.api.models.configuration.NodeAddress;
+import glide.api.models.exceptions.ClosingException;
+import glide.api.models.exceptions.ConnectionException;
+import glide.api.models.exceptions.ExecAbortException;
+import glide.api.models.exceptions.TimeoutException;
+
+import java.util.concurrent.CancellationException;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+
+public class StandaloneExample {
+
+    /**
+     * Creates and returns a GlideClient instance. This function initializes a
```

Comment:
Do we want to wrap all occurrences of the following in code tags?
- GlideClient
- GlideClientConfiguration
- ExecutionException

Reply:
I have added `@throws` tags for each exception throws from `createClient()` and `appLogic()`.

Reply:
I was referring to the `<code>` blocks